spec.bs

```diff
@@ -8349,6 +8665,104 @@ scoring signals requests=] |entriesToBatch|:
 
 <div algorithm>
 
+To <dfn>build trusted key value scoring signals request body</dfn> given a non-empty [=list=] of
+[=trusted scoring signals requests=] |entriesToBatch|:
+
+1. Let |firstRequest| be |entriesToBatch|[0].
+1. Let |compressionGroups| be an empty [=list=], whose [=list/items=] are [=scoring compression groups=].
+1. Let |compressionGroupMap| be an empty [=map=], where the [=map/keys=] are integers as
+  [=scoring compression group/compression group id=], and the [=map/values=] are [=maps=] with integers as [=map/keys=]
+  for [=scoring partition/id=], and [=scoring partitions=] as their [=map/values=].
+1. Let |compressionIdMap| be an empty [=map=], whose [=map/keys=] are [=tuples=] of ([=origin=],
+  [=origin=]) and [=map/values=] are integers.
+1. Let |renderUrlIdMap| be an empty [=map=], whose [=map/keys=] are [=URLs=] and [=map/values=] are [=tuples=]
+  of (interger, integer).
+1. Let |nextCompressionGroupId| be 0.
+1. [=map/For each=] |request| of |entriesToBatch|:
+  1. Let |mapKey| be a tuple of (|request|'s [=trusted scoring signals request/owner origin=],
+    |request|'s [=trusted scoring signals request/joining origin=]).
+  1. If |compressionIdMap| does not [=map/contain=] |mapKey|:
+    1. Set |compressionIdMap|[|mapKey|] to |nextCompressionGroupId|.

```

Comment:
Just find that you were using "set" instead of [=map/set=] when setting a map[key]'s value multiple times, in this algorithm. Please read through and fix them.
```suggestion
    1. [=map/Set=] |compressionIdMap|[|mapKey|] to |nextCompressionGroupId|.
```